upstairs/src/lib.rs

```diff
@@ -104,11 +104,15 @@ pub const IO_OUTSTANDING_MAX_JOBS: usize = 10000;
 #[async_trait]
 pub trait BlockIO: Sync {
     async fn activate(&self) -> Result<(), CrucibleError>;
+    async fn activate_with_gen(&self, gen: u64) -> Result<(), CrucibleError>;
```

Comment:
Eventually we want to get rid of `activate_with_gen` - for now, should we mark this as test-only, or hide it behind a flag? I'm also fine not doing this, so consider this a non-blocking suggestion :)

Reply:
I want to focus on getting rid of it!